src/net/gateway.ts

```diff
@@ -23,6 +22,9 @@ export type IGateway = InstanceType<typeof Gateway>
 export class Gateway {
     protected dvote: DVoteGateway = null
     protected web3: Web3Gateway = null
+    protected archiveIpnsId: string
+    protected environment: VocdoniEnvironment
+    protected network: EthNetworkID
```

Comment:
The network ID is known via the web3 provider. I think we don't need to store this here